buildkitd/Earthfile

```diff
@@ -12,7 +12,8 @@ buildkitd:
             ARG BUILDKIT_BASE_IMAGE=$BUILDKIT_PROJECT+build
         END
     ELSE
-        ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:fd74555d900aee79f0ff23c247911d8e90caa45d+build
+        ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:778351b6e8fe544cf8edd9af77728cc260213976+build
+        ARG BUILDKIT_BASE_IMAGE=github.com/earthly/buildkit:778351b6e8fe544cf8edd9af77728cc260213976+build
```

Comment:
what's going on here? why is this line duplicated?

Reply:
I just resolved a merge conflict, apparently not very well. Thanks for noticing , will fix

Reply:
this git commit needs to be updated to reference the merged commit: https://github.com/earthly/buildkit/commit/3b049bfd9c25598450037d0911255f9e3ba8d0ed